client/src/pages/DirectoryPage/DirectoryGroups.js

```diff
@@ -14,223 +14,196 @@ import picture1 from "../../assets/images/card_small1.png";
+            <Pill
+              className="Pill"
+              text={"Traveling"}
+              size={"wide"}
+              color={"pink"}
+              icon={"white"}
+              onClick={pillClick}
+              selectId={0}
+            />{" "}
+            <Pill
+              className="Pill"
+              text={"Traveling"}
+              size={"wide"}
+              color={"pink"}
+              icon={"white"}
+              onClick={pillClick}
+              selectId={0}
+            />{" "}
+            <Pill
+              className="Pill"
+              text={"Traveling"}
+              size={"wide"}
+              color={"pink"}
+              icon={"white"}
+              onClick={pillClick}
+              selectId={0}
             />
-          </div>{" "}
-          <div className="CommunityPage__groupCard-single">
-            <GroupCard
-              name="DIY with your kids"
-              members="98 Members"
-              location="Mixed Locations"
-              picture={picture1}
+            <Pill
+              className="Pill"
+              text={"Traveling"}
+              size={"wide"}
+              color={"pink"}
+              icon={"white"}
+              onClick={pillClick}
+              selectId={0}
             />
-          </div>
-          <div className="CommunityPage__groupCard-single">
-            <GroupCard
-              name="Moms traveling to Hawaii"
-              members="5"
-              location="Chicago, IL"
-              picture={picture2}
+            <Pill
+              className="Pill"
+              text={"Traveling"}
+              size={"wide"}
+              color={"pink"}
+              icon={"white"}
+              onClick={pillClick}
+              selectId={0}
             />
-          </div>
-          <div className="CommunityPage__groupCard-single">
-            <GroupCard
-              name="Group Name"
-              members="# of "
-              location="Location, State"
-              picture={picture3}
+            <Pill
+              className="Pill"
+              text={"Traveling"}
+              size={"wide"}
+              color={"pink"}
+              icon={"white"}
+              onClick={pillClick}
+              selectId={0}
             />
           </div>{" "}
-          <div className="CommunityPage__groupCard-single">
-            {" "}
-            <Card outline size="medium-wide-directory">
-              <a href="/">Discover New Groups</a>
-            </Card>{" "}
+          <a className="seeMore-Button">See More</a>
+          <div>
+            <header className="CommunityPage__header">Groups in WASHINGTON, DC:</header>
+          </div>
+          <div className="CommunityPage__groups-in-city-container">
+          {groups.map(group =>{
+            console.log(group)
+            let memberLength = 1;
+            if (memberLength){
+              const memberLength = group.members.length;
+            }
```

Comment:
This is odd - you're assigning `memberLength` a truthy value and then testing if it's truthy immediately afterwords

Reply:
Then recreating the variable